docs/src/man/subsets.md

```diff
@@ -6,7 +6,7 @@ The `DataArray` type is meant to behave like a standard Julia `Array` and tries
 
 One dimensional `DataArray`:
 
-```julia
```

Comment:
With the "julia", Lapidary tries to treat these as scripts to run and test, and they break.